exercises/hello-world/makefile

```diff
@@ -4,10 +4,17 @@ CFLAGS += -Wextra
 CFLAGS += -pedantic
 CFLAGS += -Werror
 
+VFLAGS  = --quiet
+VFLAGS += --tool=memcheck
```

Comment:
This is the default, not needed, but does not hurt.

Reply:
I like to be explicit with options for stuff students use. Saying that I've no strong feelings on it and would be happy to take it out if you like?

Reply:
My vote is to leave it in